java/README.md

```diff
@@ -14,6 +13,24 @@ Signer.from(kid, privateKey)
         .sign();
 ```
 
+### Verifying webhooks
+The `Verifier.verifyWithJwks` function may be used to verify `Tl-Signature` header signatures.
+
+```java
+// `jku` field is included in webhook signatures
+String jku = Verifier.extractJku(webhookSignature);
+
+// fetch jwks JSON from the `jku` url (not provided by this lib)
+String jwks = fetchJwks(jku):
+
+Verifier.verifyWithJwks(jwks)
+        .method("POST")
+        .path(path)
+        .headers(headers)
+        .body(body)
+        .verify(webhookSignature);
+```
+
```

Comment:
No scala and kotlin examples?

Reply:
tbf it might be simpler to have just one anyway